server/data/data-cassandra/src/main/java/org/apache/james/user/cassandra/CassandraUsersRepository.java

```diff
@@ -186,12 +186,6 @@ public int countUsers() throws UsersRepositoryException {
             .iterator();
     }
 
-    @Override
```

Comment:
yet another case of "I removed that code in the middle of something else". What's wrong with this code ?